test/Microsoft.Azure.ServiceBus.UnitTests/ExpectedMessagingExceptionTests.cs

```diff
@@ -12,13 +13,20 @@ namespace Microsoft.Azure.ServiceBus.UnitTests
 
     public class ExpectedMessagingExceptionTests
     {
-        [Fact]
-        async Task MessageLockLostExceptionTest()
+        public static IEnumerable<object> TestConnectionStrings => new object[]
+        {
```

Comment:
I realize you have changed this across all the tests. Sorry for not being more explicit with my testing comment. This changes only the transport layer. Behind the scenes, both transport types will reach the same protocolHeader, and will be parsed exactly same. In short, we do NOT need to test entire functionality again with both TCP and websockets. Only one E2E test which sends a message and receives it should be fine. 
If these were unit tests, i would have left them as-is. But given that these are E2E tests which take quite some time to run, I'll have to ask you to revert these test changes.